app/src/tasks/TaskDetail/ClientConfig.tsx

```diff
@@ -43,34 +44,29 @@ function TsClientConfig() {
     helperAggregator: Promise<Aggregator>;
   };
 
-  const taskConfig = usePromiseAll(
-    [task, leaderAggregator, helperAggregator],
-    ([task, leader, helper]) => ({
-      ...task.vdaf,
-      id: task.id,
-      leader: leader.dap_url,
-      helper: helper.dap_url,
-      timePrecisionSeconds: task.time_precision_seconds,
-    }),
-    {
-      type: null,
-      id: null,
-      leader: null,
-      helper: null,
-      timePrecisionSeconds: null,
-    } as object,
-  );
-
-  const taskArgs = JSON.stringify(taskConfig, null, 2).replace(
-    /"([^"]+)":/g,
-    "$1:",
-  );
+  // We aren't using usePromiseAll due to a bug when using Promise.all, React.useMemo(), and
```

Comment:
nit: phrase this as a TODO to return to using `usePromiseAll` once the relevant issue is fixed.